operator/v1/types_machineconfiguration.go

```diff
@@ -494,3 +494,19 @@ const (
 	// Special represents an action that is internal to the MCO, and is not allowed in user defined NodeDisruption policies.
 	SpecialStatusAction NodeDisruptionPolicyStatusActionType = "Special"
 )
+
+// These strings will be used to indicate success and failures for MachineConfiguration knobs via conditions.
+// Please note: These are subject to change. Both additions and deletions may be made.
+const (
+	// MachineConfigurationBootImageUpdateFailed means that the MCO ran into an error while reconciling boot images.
+	// +openshift:enable:FeatureGate=ManagedBootImages
+	MachineConfigurationBootImageUpdateFailed string = "BootImageUpdateFailed"
+
+	// MachineConfigurationBootImageUpdateSuccess means that the MCO was able to successfully update all boot images.
```

Comment:
is there a generation involved here?  If a new bootimage is created, this immediately becomes false?

Reply:
I know we've removed the "Success" condition as it is seemed redundant, but for the 2 remainder states the generation of the boot images configmap will be included in every condition update as we discussed